api/package.json

```diff
@@ -1,15 +1,10 @@
 {
   "name": "ezbids",
-  "version": "1.1.0",
+  "version": "1.0.0",
   "description": "",
-  "main": "index.js",
   "scripts": {
-    "uidev": "cd ui && lite-server",
-    "tsc": "tsc",
-    "dev": "ts-node-dev --respawn --transpileOnly ./api/ezbids.ts",
-    "prod": "tsc && node ./build/ezbids.js",
-    "deploy": "ssh -t hayashis@test.brainlife.io \"sudo su - -c 'cd /root/docker/ezbids && ./update.sh'\"",
-    "deploy-prod": "ssh -t -J hayashis@brainlife.io ubuntu@ezbids \"sudo su - -c 'cd /root/docker/ezbids && ./update.sh'\""
+    "dev": "ts-node-dev --respawn --transpile-only ezbids.ts",
+    "prod": "tsc && node ./build/ezbids.js"
```

Comment:
are files transpiled into JS when built? Just double checking that the prod script here was intentional and shouldn't instead be: `"prod": "tsc && node ./build/ezbids.ts"`